### ISSUE TITLE
Ajouter la CI

### ISSUE THREAD

**Issue by AdrienRedon** [psegalen/stdrenegade#2]
Il faudrait ajouter la CI (au moins le lint pour gagner du temps sur les PR) 🙂

**Comment by psegalen** [comment]
Yes, on fera ça en live un de ces jours, ça peut intéresser du monde !
Pour le lint oui, on ajoutera ESLint très bientôt ! ;)
Merci !
